src/inputs/WizKeyValue.tsx

```diff
@@ -11,7 +11,11 @@ type KeyValueProps = InputCommonProps & { placeholder?: string; summaryList?: bo
 
 export function WizKeyValue(props: KeyValueProps) {
     const { displayMode: mode, value, setValue, hidden, id } = useInput(props)
-    const [pairs] = useState<{ key: string; value: string }[]>(() => Object.keys(value).map((key) => ({ key, value: value[key] })))
+    let pairs = [{ key: '', value: '' }]
+    if (value instanceof Object) {
+        pairs = Object.keys(value).map((key) => ({ key, value: value[key] }))
+    }
+
```

Comment:
@dhaiducek I think you're right - looks like the only purpose of `useState` here was to prevent it from getting updated if the value changes... @jamestalton is that right? Why was that desired?

Reply:
i'll have to take a look

Reply:
@jamestalton Have you had a moment to take a look at the use of `useState` here?

Reply:
@dhaiducek This looks fine. If the value changes in the YAML, `pairs` will now be recomputed. You could memoize `pairs` using the `useMemo` hook, but It is probably not very expensive to compute on each render. Is this fix intended for ACM 2.6?